index.html

```diff
@@ -26,8 +26,10 @@
 	<p id="resolutions">Common: 
 		<a href="#" tabindex="-1">1920×1080</a>
 		<a href="#" tabindex="-1">1680×1050</a>
+		<a href="#" tabindex="-1">1680×900</a>
 		<a href="#" tabindex="-1">1440×900</a>
 		<a href="#" tabindex="-1">1366×768</a>
+		<a href="#" tabindex="-1">1360×768</a>
```

Comment:
I would be opposed to adding new predefined resolutions that are so similar to existing examples. But even if I wasn't, it's always a good practice to send separate PRs for separate changes.